app/components/__test__/Button.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import { Button, LinkButton } from '../Button';
+
+describe('Button', () => {
+  const btnText = 'Click me';
+  it('renders', () => {
+    const { getByText } = render(
+      <Button size="small" label="Click me" varient="primary" disabled={false} />
+    );
+    expect(getByText(btnText)).toBeInTheDocument();
+  });
+
+  it('renders with different sizes', () => {
+    const { getByText } = render(
+      <Button size="small" label="Click me" varient="primary" disabled={false} />
+    );
+    expect(getByText(btnText)).toHaveClass('py-1 px-2 text-sm gap-1');
+    expect(getByText(btnText).classList.contains('border-transparent')).toBeTruthy();
```

Comment:
checking individual classes on components would be redundant we can use snapshot tests for this.

Reply:
Currently, we are just creating UTs. We can migrate later on.